excel.go

```diff
@@ -158,20 +159,29 @@ func syncPasswordManagerUsersToMACFinUsers(f *excelize.File, input *Input) error
 		rowToDelete := toSheetCoord(rowsToDelete[idx] + rowOffset)
 		err := f.RemoveRow(automatedSheet, rowToDelete)
 		if err != nil {
-			return fmt.Errorf("failed removing row %d from %s sheet in file %s: %s", rowToDelete, automatedSheet, input.Filename, err)
+			return fmt.Errorf("failed removing row %d from %s sheet in file %s: %s", rowToDelete, automatedSheet, f.Path, err)
 		}
 	}
 
 	err = f.Save()
 	if err != nil {
-		return fmt.Errorf("failed saving %s after synchronizing automated sheet users to MACFin users: %s", input.Filename, err)
+		return fmt.Errorf("failed saving file %s after synchronizing automated sheet users to MACFin users: %s", f.Path, err)
 	}
 
 	err = sortRows(f, input, automatedSheet)
 	if err != nil {
 		return fmt.Errorf("failed sorting %s after synchronizing sheet to MACFin users: %s", automatedSheet, err)
 	}
 
+	if numRows > initialNumRows || len(rowsToDelete) > 0 {
```

Comment:
I thought this was going to be unconditional? just seems simpler not to have to be sure that the conditions are correct

Reply:
oh, ok. It is now unconditional.